test/unit/test_assembly.py

```diff
@@ -353,7 +353,7 @@ def test_gapfill(self):
         in_scaffold = join(inDir, 'TestOrderAndOrient', 'expected.ebov.doublehit.fasta')
         with util.file.tempfname(suffix='.filled.fasta') as filled:
             assembly.gapfill_gap2seq(in_scaffold=in_scaffold,
-                                     in_bam=join(inDir, 'G5012.3.testreads.bam'), out_scaffold=filled)
+                                     in_bam=join(inDir, 'G5012.3.testreads.bam'), out_scaffold=filled, threads=1, random_seed=23923937)
```

Comment:
Can you also add explicit unit tests that test:
 - empty fasta input (no contigs)
 - empty bam input (no reads, just headers)

Also, do we know that gap2seq works with both single and paired reads?

Reply:
I think this version doesn't use read pairing information, but the newsest one does (to localize reads to gaps).  We could use set of references to better localize reads to gaps.